lti_consumer/models.py

```diff
@@ -63,21 +64,32 @@ class LtiConfiguration(models.Model):
     )
 
     # Configuration storage
-    # Initally, this only supports the configuration
-    # stored on the block, but should be expanded to
-    # enable storing LTI configuration in this model.
+    # Initally, this only supported the configuration
+    # stored on the block. Now it has been expanded to
+    # enable storing LTI configuration in the model itself or in an external
+    # configuration service fetchable using openedx-filters
     CONFIG_ON_XBLOCK = 'CONFIG_ON_XBLOCK'
     CONFIG_ON_DB = 'CONFIG_ON_DB'
+    CONFIG_EXTERNAL = 'CONFIG_EXTERNAL'
     CONFIG_STORE_CHOICES = [
         (CONFIG_ON_XBLOCK, _('Configuration Stored on XBlock fields')),
         (CONFIG_ON_DB, _('Configuration Stored on this model')),
+        (CONFIG_EXTERNAL, _('Configuration Stored on external service')),
     ]
     config_store = models.CharField(
         max_length=255,
         choices=CONFIG_STORE_CHOICES,
         default=CONFIG_ON_XBLOCK,
     )
 
+    # ID of the configuration if the configuration is obtained from the
+    # external service using filters
+    external_id = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True
```

Comment:
Django will set this as an empty string by default, no need for the null value here.
```suggestion
```

Reply:
@giovannicimolin Removing the `null=True` trips up a lot of the tests with 
```
django.db.utils.IntegrityError: NOT NULL constraint failed: lti_consumer_lticonfiguration.external_id
```
So, I have left it unchanged.